view/scripts/app.js

```diff
@@ -10,17 +10,7 @@ const view = new View(".action-bar",".table-wrapper", charts, popup)
 const data = new Data("http://localhost:3000")
 const controller = new Controller(view, data)
 
+//Todo - We could put this also in the controller
```

Comment:
Only app.js is directly loaded into the index.html file, so it makes sense to trigger the load event of the window within the app.js script. Also it's easy to see the entrypoint of the application.